placement/algo/helper.go

```diff
@@ -227,7 +229,7 @@ func (ph placementHelper) getShardLen() int {
 	return len(ph.uniqueShards)
 }
 
-func (ph placementHelper) canAssignRack(shard uint32, from placement.HostShards, toRack string) bool {
+func (ph placementHelper) CanMoveShardToRack(shard uint32, from placement.HostShards, toRack string) bool {
```

Comment:
Will rename, but in that case I will not do this check if the constraint is loosened